### DISCUSSION TITLE
NAV - the navigation data are not enabled

### DISCUSSION THREAD

**Discussion by Silintone** [Mr-MIBonk/M.I.B._More-Incredible-Bash#132] [General]
Good day

I updated my firmware on my mhi2 to do the sub things from harman and bruce , all is well there.
I then attempted the fec codes for nav database which was successful and showing as valid, i then updated my maps and now it shows maps 2023 
but after 1 min in the nav  i get the error. *the navigation data are not enabled*
How would i go about validating this ?

Unit is MHI2_ER_AUG22_K2161  ( cant update as the new FW says my train is not supported ) 

 **UPDATE** updated to latest 1438 / 3348 FM

**Comment by harman-f** [comment]
Show your FECs

**Comment by Silintone** [comment]
Found the issue to be the 023000ee fec is EU maps lifetime maps 
My map database is ROW 
So i manually created 023d0030 and patch that 

All is now working on the maps front :)

**Reply by paulinhapirolas** [reply]
why did you created it manually? you opened the fec.txt and change it?

**Comment by paulinhapirolas** [comment]
Hi,after install the patch, the 023000ee fec says that is ilegal but i can install the 2022/23 maps, but the problem is  that the nav last 30 secs and turns off and give the error "the navigation database are not enabled"
what i did wrong?
my firmware are the MHI2_ER_AUG22_K3346_MU1438

**Comment by paulinhapirolas** [comment]
My Fecs are:
![IMG_20230321_222738](https://user-images.githubusercontent.com/128599265/226934469-04790560-34cc-4d1b-97c5-c5f8a5ef5de5.jpg)
![IMG_20230321_222745](https://user-images.githubusercontent.com/128599265/226934556-28af996a-987f-4619-8f36-64cdee638ed4.jpg)

**Comment by Silintone** [comment]
Well you might be in luck bud, that fec is for EU maps, your likely under the ROW ( rest of the world) I manually made up my own fec code for that.
If you are on ROW let me know then I'll explain

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by paulinhapirolas** [comment]
so, how can i proceed to solve my problem. Many Thanks

**Comment by Silintone** [comment]
I first need you to confirm your region is ROW. Don't want you to waste time updating more fec codes and they fail

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by paulinhapirolas** [comment]
I'm not with the car right now but in 3 hours I tell you.

**Comment by Silintone** [comment]
Well to save you time , only do this if your maps are ROW
The files on the sd card , edit the fec list  txt file.
replace the 023000ee   with   023d0030 

Then re run the sd card fec patch in MIB and reboot.
It will work then .
Maps stopping after 30 seconds is the same issue i had

**Reply by mustafa-tolu** [reply]
Thanks for the approach. Maybe we can add "_Using fresh MIB copy on reformatted SD_" step. (**Sources**: https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/discussions/96#discussioncomment-4914123 **and** https://mibwiki.one/doc/navigation-gdV1YkhdTO)

**Comment by paulinhapirolas** [comment]
Hi, here my region. I think is EU
![IMG_20230323_072311](https://user-images.githubusercontent.com/128599265/227133011-ce52f9de-c670-4d58-8d96-04b3c0623145.jpg)

**Reply by paulinhapirolas** [reply]
Another picture
![IMG_20230323_075058](https://user-images.githubusercontent.com/128599265/227137970-721d0cf5-2897-417f-9528-60806a9bed6e.jpg)

**Comment by harman-f** [comment]
Share your log file

**Reply by paulinhapirolas** [reply]
Which log file?

**Comment by paulinhapirolas** [comment]
So you suggest that I change this doc
![IMG_20230323_084840](https://user-images.githubusercontent.com/128599265/227150449-47b2e921-4582-44c5-a1df-0cecac512110.jpg)

**Comment by Silintone** [comment]
You fall under EU so yours is correct, don't use the one I suggested.
Reboot the mmi and re run it.
But your addfec file doesn't look right.
They should be listed top to bottom. Not all in one string.
Try re-download the mib and copy to flash and test that

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by paulinhapirolas** [reply]
And now?
![IMG_20230323_092329](https://user-images.githubusercontent.com/128599265/227159000-bc5c61ab-76ec-4c72-9053-bebe5c9caa45.jpg)

**Comment by paulinhapirolas** [comment]
Should I put the number of EU maps 2023 in the adfec. Txt?

**Comment by Silintone** [comment]
The list now looks correct .
Yes that .......EE needs to be there , then go in MIB on the car and install it.
I was able to install it but because my car is ROW region its sitting there for nothing, thats when i had to make a plan for the ROW FEC.
You shouldnt have any issues now
________________________________

**Reply by paulinhapirolas** [reply]
Hi, when I separate the codes didn't recognize the codes when install the fec. All togheter recognized the code and the 023000ee stays LEGAL but when the map turns off by herself the 023000ee turns ILEGAL. So for now no solution
